pkg/interface/src/views/apps/profile/components/EditProfile.tsx

```diff
@@ -103,28 +144,73 @@ export function EditProfile(props: any): ReactElement {
         initialValues={contact || emptyContact}
         onSubmit={onSubmit}
       >
-      <Form width="100%" height="100%" p={2}>
-        <Input id="nickname" label="Name" mb={3} />
-        <Col width="100%">
-          <Text mb={2}>Description</Text>
-          <MarkdownField id="bio" mb={3} s3={props.s3} />
-        </Col>
-        <ColorInput id="color" label="Sigil Color" mb={3} />
-        <Row mb={3} width="100%">
-          <Col pr={2} width="50%">
-            <ImageInput id="cover" label="Cover Image" s3={props.s3} />
-          </Col>
-          <Col pl={2} width="50%">
-            <ImageInput id="avatar" label="Profile Image" s3={props.s3} />
-          </Col>
-        </Row>
-        <Checkbox mb={3} id="isPublic" label="Public Profile" />
-        <GroupSearch label="Pinned Groups" id="groups" groups={props.groups} associations={props.associations} publicOnly />
-        <AsyncButton primary loadingText="Updating..." border mt={3}>
-          Submit
-        </AsyncButton>
-      </Form>
-    </Formik>
-  </>
+        {({ setFieldValue }) => (
+          <Form width='100%' height='100%'>
+            <ProfileHeader>
+              <ProfileControls>
+                <Row>
+                  <button
+                    type='submit'
+                    style={{ appearance: 'none', background: 'transparent' }}
```

Comment:
@matildepark This empty, unstyled submit button with an Indigo Text element inside of it gets us what we want visually and interactively, but is probably bad form. Do you have suggestions on how to submit a Formik form with two submit buttons? I tried calling onSubmit directly but to no avail.

Reply:
This is a confusing question, because [button elements submit forms by default](https://stackoverflow.com/questions/3314989/can-i-make-a-button-not-submit-a-form). Both buttons should!

Reply:
```
                  <Button
                    type='submit'
                    display='inline'
                    cursor='pointer' fontWeight='500' color='blue' pl='0' pr='0'
                    border='0'
                    style={{ appearance: 'none', background: 'transparent' }}
                  >
                    Save Edits
                  </Button>
```

<img width="282" alt="image" src="https://user-images.githubusercontent.com/20846414/109747011-638d7500-7ba4-11eb-8cd4-d5201bee1909.png">

Reply:
Thanks, wasn't sure what the policy was on overriding buttons vs using text with click handlers.